src/SpatialUnitTests/Euclidean/Ray3DTests.cs

```diff
@@ -56,20 +55,5 @@ public void XmlTests(string ps, string vs, bool asElements, string xml)
             var ray = new Ray3D(Point3D.Parse(ps), UnitVector3D.Parse(vs));
             AssertXml.XmlRoundTrips(ray, xml, (e, a) => AssertGeometry.AreEqual(e, a, 1e-6));
         }
-
-        [Test]
-        public void BinaryRoundtrip()
-        {
-            var v = new Ray3D(new Point3D(1, 2, -3), UnitVector3D.Create(1, 2, 3));
-            using (var ms = new MemoryStream())
-            {
-                var formatter = new BinaryFormatter();
-                formatter.Serialize(ms, v);
-                ms.Flush();
-                ms.Position = 0;
-                var roundTrip = (Ray3D)formatter.Deserialize(ms);
-                AssertGeometry.AreEqual(v, roundTrip);
-            }
-        }
```

Comment:
Can't we keep the binary round trips in .Net Framework builds?

Reply:
I'm not sure if you are thinking that I just removed them or you are objecting to me moving them.  They are still there, just they all live in BinaryFormatterTests.cs and are wrapped in an #if NETCOREAPP1_1 == false.  So they still run for both netcoreapp2.0 and net45.

If you are asking about me moving them then a few thoughts on this:
 - serialization as a set be it, binary, xml, json, protobuf, etc is a transversal concept.  If we support a particular serialization then it should work on all types in the library.  It is easier to track and ensure this by keeping the serialization pieces for a given target together.
- a given serialization is not particularly tied to the functionality of the type, it would be odd to have legacy binary serialization with the type but other serialization tests apart.  If they are all with the types then every unit test class would have non core dependencies
- having framework version specific support is easier to manage if there is a minimum of #if scattered through the code.  By grouping by serialization testing together, those checks can be skipped on mass for platforms where they are not supported

Reply:
No, I was indeed missing the place - thanks for clarifying!